packages/dendron-cli/src/commands/publishCLICommand.ts

```diff
@@ -156,20 +149,48 @@ export class PublishCLICommand extends CLICommand<CommandOpts, CommandOutput> {
     const { cmd } = opts;
     const ctx = "execute";
     this.L.info({ ctx });
+    const spinner = ora();
     try {
       switch (cmd) {
         case PublishCommands.INIT: {
-          return await this.init(opts);
+          return await this.init({ ...opts, spinner });
         }
         case PublishCommands.BUILD: {
           return this.build(opts);
         }
         case PublishCommands.DEV: {
+          const isInitialized = await NextjsExportPodUtils.isInitialized({
```

Comment:
instead of passing in ora, can we just call `ora.start` before and after this function is initialized?

this makes NextjsExportUtils not have to be aware of the progress indicator and allows us to switch it out between CLI and Plugin

Reply:
I agree. The things we run are chopped up into atomic enough operations that we don't really need intermediate prints within the operation anymore. In fact everything on the plugin side already has withProgress wrapped around and pods-core knows nothing about it.